src/src/Commands/InternalCommands.cpp

```diff
@@ -416,11 +416,15 @@ bool executeInternalCommand(command_case_data & data)
     #endif // ifdef FEATURE_SD
 
       if (data.cmd_lc[1] == 'e') {
+        #ifdef FEATURE_ESP_P2P
         COMMAND_CASE_A(    "sendto", Command_UPD_SendTo,      2); // UDP.h    // FIXME TD-er: These send commands, can we determine the nr
                                                                   // of
                                                                   // arguments?
+        #endif // FEATURE_ESP_P2P
         COMMAND_CASE_A("sendtohttp", Command_HTTP_SendToHTTP, 3); // HTTP.h
+        #ifdef FEATURE_ESP_P2P
         COMMAND_CASE_A( "sendtoudp", Command_UDP_SendToUPD,   3); // UDP.h
+        #endif
```

Comment:
I think `SendToUdp` is not involved in the P2P protocol, but can be used for UDP communication to any IP address.